tests/v2v_test.go

```diff
@@ -0,0 +1,55 @@
+package tests_test
+
+import (
+	"log"
+	"os"
+
+	. "github.com/onsi/ginkgo"
+	"kubevirt.io/kubevirt-ansible/tests"
+)
+
+// template parameters
+const (
+	v2vServiceInstance    = "tests/manifests/import-vm-v2v-apb.yml"
+	dstv2vServiceInstance = "/tmp/test-v2v.json"
+)
+
+// Array of enviroment variables to get from testing env
+var listofEnvVars = []string{"ocpID", "ocpPass", "VMWareURL", "VMName", "VMWareID", "VMWarePass"}
+
+// Map to store Enviroment variables
+var v2vEnvVars map[string]string
+
+// Get all the Env variable needed for tests
+func getV2vEnv() {
+	v2vEnvVars = map[string]string{}
+	for _, vars := range listofEnvVars {
+		val, ok := os.LookupEnv(vars)
+		if !ok {
+			//raise exception
```

Comment:
raise exception?

Reply:
the env vars are important for the test, so `log.fatal` becomes important